src/helpers/issue.ts

```diff
@@ -301,3 +317,18 @@ export const getAssignedIssues = async (username: string) => {
 
   return assigned_issues;
 };
+
+export const getOpenedPullRequests = async (username: string) => {
```

Comment:
This method returns opened PRs with no reviews and created within the last 24 hours. Naming should be better, smth like `getLast24HoursOpenedPullRequestsWithNoReviews` but more consise.